data_collection/gazette/spiders/rj/rj_sao_joao_de_meriti.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
+
+import scrapy
+from dateutil.rrule import MONTHLY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjSaoJoaoDeMeritiSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3305109"
+    name = "rj_sao_joao_de_meriti"
+    allowed_domains = ["niteroi.rj.gov.br"]
+    start_urls = ["https://transparencia.meriti.rj.gov.br/"]
+    BASE_URL = "https://transparencia.meriti.rj.gov.br/ver20230623/WEB-ObterAnexo.rule?sys=LAI&codigo={ATTACHMENT_CODE}"
+    start_date = dt.date(2017, 1, 1)
+    custom_setting = {"DOWNLOAD_DELAY": 1.5}
+
+    def start_requests(self):
+        url = "https://transparencia.meriti.rj.gov.br/diario_oficial_get.php"
+        initial_date = self.start_date
+
+        for monthly_date in rrule(
+            freq=MONTHLY, dtstart=initial_date, until=self.end_date
+        ):
+            month_year = monthly_date.strftime("%m/%Y").lstrip("0")
+            yield scrapy.FormRequest(
+                url=url,
+                formdata={"mesano": month_year},
+            )
+
+    def parse(self, response):
+        for gazette_data in response.json():
+            raw_gazette_date = gazette_data["Data_Formatada"]
+            gazette_date = dt.datetime.strptime(raw_gazette_date, "%d/%m/%Y").date()
+            if gazette_date < self.start_date or self.end_date < gazette_date:
+                continue
```

Comment:
Uma dica pythonica é que a verificação pode ser feita de uma vez só:
```suggestion
            if self.end_date < gazette_date < self.start_date:
                continue
```

Reply:
@almeidadm, você chegou e executar uma nova extração por período (`start_date` e `end_date` definidos) após essa modificação?

Reply:
Oi, @AlexJBSilva!
Realmente não tinha testado essa solução.
Corrigi agora no último commit que realizei :)

Reply:
Ah, ótimo! 
Realmente eu tinha ficado em dúvida se precisaria ser maior igual ou apenas maior.
Fiz as alterações e os testes, e agora parece que agora esta com o comportamento esperado!.